manage.py

```diff
@@ -816,18 +818,18 @@ def _add_lambda_deploy_subparser(subparsers):
             help=ARGPARSE_SUPPRESS
         )
 
-    # flag to manually bypass rule staging for specific rules during deploy
+    # flag to manually demote specific rules to staging during deploy
     lambda_deploy_parser.add_argument(
-        '--unstage-rules',
+        '--stage-rules',
```

Comment:
I don't think precedence of argument `--stage-rules` and `--unstage-rules` matter. Isn't it?